docs/community/training/debugging/manifesto.md

```diff
@@ -0,0 +1,12 @@
+# Debugging manifesto
+
+<figure markdown="span">
```

Comment:
We could consider listing a few errors that we have found ourselves around here? 
Something that springs to mind is an "error" someone came to me with in a library, but in reality they didnt read the documentation and they thought an error code of 1 was good.